MapleServer2/Managers/PartyManager.cs

```diff
@@ -16,9 +16,10 @@ public void AddParty(Party party)
         PartyList.Add(party.Id, party);
     }
 
-    public void RemoveParty(Party party)
+    public void RemoveParty(Party? party)
     {
-        PartyList.Remove(party.Id);
+        PartyList.Remove(party?.Id ?? -1);
```

Comment:
If the party is null, should we not just return here instead?

Reply:
Party should never be null here anyways, leftovers.